src/Util/UtilTrait.php

```diff
@@ -65,6 +65,11 @@ protected function getOperationsPath(InputInterface $input, $default = 'Migratio
         $folder = $input->getOption('source') ?: $default;
 
         $dir = ROOT . DS . 'config' . DS . $folder;
+
+        if (defined('CONFIG')) {
+            $dir = CONFIG . $folder;
```

Comment:
My point was that this if is not necessary then anymore, if it always is defined.
But no big deal :)

Reply:
The CONFIG constant is not always going to exist in all host applications.